modules/imap/js_modules/route_handlers.js

```diff
@@ -51,4 +115,12 @@ function applyImapMessageContentPageHandlers(routeParams) {
     if (window.githubMessageContentPageHandler) githubMessageContentPageHandler(routeParams);
     if (window.pgpMessageContentPageHandler) pgpMessageContentPageHandler();
     if (window.wpMessageContentPageHandler) wpMessageContentPageHandler(routeParams);
+}
+
+function applyImapServersPageHandlers() {
+    imap_setup_server_page();
+}
+
+function applyImapInfoPageHandlers() {
+    setTimeout(imap_status_update, 100);
```

Comment:
You added new handlers, but for which pages are they getting called?
FYI handlers are bound to appropriate pages as in https://github.com/cypht-org/cypht/blob/master/modules/core/navigation/routes.js